hw3/03-react/src/components/ChartComponent.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect, useRef } from 'react';
+import { Chart, registerables } from 'chart.js';
+
```

Comment:
I really like your implementation here, great use of the useEffect hook, and I think this is clean code.